test/integration/selenium-helpers.js

```diff
@@ -62,13 +63,21 @@ class SeleniumHelper {
         return driver;
     }
 
+    getChromeVersionNumber () {
+        let chromedriverVersion = pkg.devDependencies.chromedriver;
+        let versionFinder = /\d+\.\d+/;
+        let versionArray = versionFinder.exec(chromedriverVersion);
+        return versionArray[0];
+    }
+
     getSauceDriver (username, accessKey, name) {
+        let chromeVersion = this.getChromeVersionNumber();
```

Comment:
I think we should handle the case where this comes back `undefined` if e.g. we move `chromedriver` from `devDependencies` to `dependencies`. I think throwing a nice error message about how you couldn't find the`chromedriver` version from `package.json` would be nice.

Now that I write that — can we just import and inspect `chromedriver` for the version rather than looking at the `package.json`?

Reply:
Pulled the version of chromedriver from the package and have removed the reference to the package.json.